starport_template/lib/main.dart

```diff
@@ -16,7 +17,8 @@ void main() {
 }
 
 void _buildDependencies() {
-  StarportApp.baseEnv = BaseEnv();
+  final env = BaseEnv();
+  StarportApp.baseEnv = BaseEnvUtil(env);
```

Comment:
let's just get rid of the `String.fromEnvironment` reading as it causes too many problems.
```suggestion
  StarportApp.appConfig = AppConfig();
```

Reply:
I have removed the  `String.fromEnvironment`